src/io/functions/iologindata_load_player.cpp

```diff
@@ -828,6 +828,27 @@ void IOLoginDataLoad::loadPlayerForgeHistory(std::shared_ptr<Player> player, DBR
 	}
 }
 
+void IOLoginDataLoad::loadPlayerStoreHistory(std::shared_ptr<Player> player, DBResult_ptr result) {
+	if (!result || !player) {
+		g_logger().warn("[IOLoginData::loadPlayer] - Player or Result nullptr: {}", __FUNCTION__);
+		return;
+	}
+
+	std::ostringstream query;
+	query << "SELECT * FROM `store_history` WHERE `account_id` = " << player->getAccountId();
```

Comment:
Please prefer using fmt as stringstream.
```suggestion
	const std::string query = fmt::format("SELECT * FROM `store_history` WHERE `account_id` = {}", player->getAccountId());
```